src/formatDate.js

```diff
@@ -7,8 +7,35 @@
  *
  * @returns {string}
  */
+
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const formatedDate = [];
```

Comment:
The variable name 'formatedDate' contains a typo. The correct spelling is 'formattedDate'.